pom.xml

```diff
@@ -146,7 +146,7 @@
     <powermock.version>1.6.2</powermock.version>
     <prometheus.version>0.8.0</prometheus.version>
     <guava.version>27.0.1-jre</guava.version>
-    <parquet.version>1.10.0</parquet.version>
+    <parquet.version>1.11.0</parquet.version>
```

Comment:
Is parquet a transitive dependency of any other libraries in our codebase? i.e. any of the hadoop clients/classes

Reply:
I took a look here https://mvnrepository.com/artifact/org.apache.parquet/parquet-avro/usages?p=1 and overall seems fine.

Reply:
Can you run `mvn dependency:graph` and look for the parquet usages?

Reply:
I took a look and it looks like the only artifact that may conflict is the `parquet-hadoop-bundle` (v1.8.1) but that dependency is already on a different version anyway. So I think it's ok